### PR TITLE
Landpage

### PR DESCRIPTION
**Pull request by Artbsc1992**
# Achievements

-Created HTML and CSS files.
-Created a Landpage for my portfolio using Figma template.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <div class="status-bar">
+        <div class="clock">9:40</div>
+        <div class="status"><img src="Img/status.png" alt="wifi"></div>
+        <div class="name">Arturo Hermida</div>
+        <div class="dropdown"><img src="Img/Enabled.png" alt="drop"></div>
```
Comment (ShoiraTa) [inline index.html:18]: - [ ] `Arturo Hermida` is supposed to be a logo, and usually, logo is a link that takes users to the homepage. Please wrap Arturo Hermida with  an `<a> `tag
- [ ] Please try to match your design with the one in Figma, currently logo on your design is wrapping and is not vertically centred :
 
Your design  | Figma design
------------- | -------------
Screenshot 2022-03-17 at 10 17 31 | Screenshot 2022-03-17 at 10 17 51
Please center the content, decrease the logo font size and make sure to have same paddings

File: index.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <div class="status-bar">
+        <div class="clock">9:40</div>
+        <div class="status"><img src="Img/status.png" alt="wifi"></div>
```
Comment (ShoiraTa) [inline index.html:16]: - [ ] I recommend you to use semantic HTML, please wrap the `humburger` icon and logo into  a `nav` tag.

File: index.html (hunk 1 above)
Comment (ShoiraTa) [inline index.html:18]: - [ ] Please wrap `humburger` button with a `<button>` tag

File: style.css [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+  flex-direction: column;
+  position: absolute;
+  width: 375px;
+  height: 590px;
+  left: 0;
+  top: 92px;
+  background-color: #1c1a19;
+  justify-content: center;
+  align-items: center;
+}
```
Comment (ShoiraTa) [inline style.css:70]: > You have done an amazing job following the design guidelines, but the container is a bit small. Mobile screens can be of different sizes and start from `425 px` (mobile lg screen). While your design is good for devices starting from `375px`(mobile md screen). 
- [ ] To make the portfolio more responsive I recommend increasing the width property to `425 px`
Screenshot 2022-03-17 at 10 10 16

File: index.html [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+        <h1 class="hello">Hey there.<br>I'm Arturo.</h1>
+        <h3 class="profession">I'am a Software Developer</h3>
+        <p class="description">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don't hestiate to contact me.</p>
+        <div class="social">         
+            <a href="#" class="icon1"><img src="Img/github.png" alt="github"></a>
+            <a href="#" class="icon2"><img src="Img/linkedin.png" alt="linkedin"></a>
+            <a href="#" class="icon3"><img src="Img/icon3.png" alt="icon3"></a>
+            <a href="#" class="icon4"><img src="Img/twitter.png" alt="twitter"></a>
+            <a href="#" class="icon5"><img src="Img/icon5.png" alt="icon 5"></a>
```
Comment (ShoiraTa) [inline index.html:30]: - [ ] All the social links should be wrapped with `<ul> <li>` tags, eg.: 

```
<ul>
  <li>
    <a href="#" class="icon1"><img src="Img/github.png" alt="github"></a>
  </li>
  <li>
    <a href="#" class="icon2"><img src="Img/linkedin.png" alt="linkedin"></a>
  </li>
</ul>
```

File: index.html [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+    </div>
+    <img src="Img/design.png" alt="design" class="ilustration">
+    </div>
```
Comment (ShoiraTa) [inline index.html:33]: - [ ] This image is not a part of the content, therefore please make sure to load it as a background image from `css`.

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi @Artbsc1992 ,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear by mentioning my Github username.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 6]
```diff
@@ -1,73 +1,26 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# My Portfolio
 
-> Description the project.
+> Hello I'm Arturo Hermida and I'm a fullstack developer.
```
[the remark this answers is not in the document]
Comment (Artbsc1992) [reply README.md:5]: Thank you so much @ShoiraTa for this recommendation :smiley:

**Review by OLIPLICHE — APPROVED** [review]
Hi @Arturo, 
Good job in making the required changes 👏
## Please there are some issues which you really need to consider for your next projects. sorry that it wasn't pointed out earlier. 
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Kindly note that there should be no custom changes to linter config. Please Use this [Link](https://github.com/microverseinc/linters-config) for Linter configuration
- Please create a `.gitignore` file, and the `node_modules` folder should always be in the `.gitignore`

## STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 





Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**